deploy/02_dlend/04_periphery_post/03-ui-helpers.ts

```
@@ -21,6 +21,14 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     return true;
   }
 
+  // Use WSAGA if available, otherwise fallback to SAGA
+  const wethAddress = config.tokenAddresses.WSAGA || config.tokenAddresses.SAGA;
+
+  if (!wethAddress) {
+    console.log("Neither WSAGA nor SAGA configured for this network. Skipping UI helpers deployment.");
```

I don't think we can set up dLEND UI without the UI helpers... we should fail the deployment if SAGA address is not configured, and ignore WSAGA since SAGA is WSAGA here